toolkit/src/main/cpp/include/version.h

```diff
@@ -0,0 +1 @@
+#define GODOT_META_TOOLKIT_VERSION "1.0.2-stable"
```

Comment:
This should have the copyright notice, and `#pragma once` (or header classic header guards if we're still using those in this repo)